watchtower/wtclient/client_test.go

```diff
@@ -1466,7 +1464,7 @@ var clientTests = []clientTest{
 			// Finally, allow the tower to ack session creation,
 			// allowing the state updates to be sent through the new
 			// session.
-			h.server.Stop()
+			_ = h.server.Stop()
```

Comment:
Seems to always return nil, but maybe safer to require.NoError to be future proof. Or alternatively remove the return value.

Reply:
good call, added require checks 👍